src/components/Home.js

```diff
@@ -5,8 +5,8 @@ export default class Home extends React.Component {
   render() {
     return (
       <React.Fragment>
-        <h1>Home Page</h1>
-        <Link to="history/chest-pain">Chest Pain</Link>
+        <h1>Potential login page</h1>
+        <Link to="/history">Log in here</Link>
```

Comment:
I don't think we should remove the current flow and replace it with a login screen, at least not yet - 

1) I don't think we'll be able to demo user authentication functionality in time for the deadlines we're putting in for this week - so this would mean we're replacing a functional link with an empty one
2) I think we discussed that the app wouldn't restrict people to be logged in/registered to get their basic functionality - although I can't find this in any of our current issues so this might be misinterpretation on my part.

Reply:
We did this, cause there was a HOME component which just had one line of text in it, or something and was absolutely useless, so we decided to replace it with login, only because the route for history (which originally was "/" now needs to be "/history" for some reason, which doesnt make to much sense